---
title: "EL-DESeq2"
author: "Maggie Schedl"
date: "6/9/2020"
output: github_document
---

```{r}
library(DESeq2)
# library(apeglm) can't use this one
library(ggplot2)
# library(vsn) can't use this one
library(pheatmap)
library(RColorBrewer)
# library(genefilter) can't use this one
library(reshape2)
library(rsconnect)
library(gplots)
library(ashr)
# library(limma) can't use this one
library(dplyr)
library(tidyr)
library(dlookr)
library(tximport)
```


DESeqDataSetFromMatrix
http://bioconductor.org/packages/devel/bioc/vignettes/DESeq2/inst/doc/DESeq2.html

```{r}
# input sample/treatment info
Sample_info <- read.csv("el-treatment-data.csv", header=TRUE, sep=",")
print(Sample_info)

```

Shouldn't be using the abundance estimation method from trinity. Should be using tximport program, that takes the RSEM results from each sample
"Import and summarize transcript-level abundance estimates for transcript- and gene-level analysis"


https://bioc.ism.ac.jp/packages/3.5/bioc/vignettes/tximport/inst/doc/tximport.html#import-transcript-level-estimates
https://bioconductor.org/packages/devel/bioc/vignettes/tximport/inst/doc/tximport.html#Import_transcript-level_estimates

Have copy to my computer all of the RSEM.genes.results files because this is what it takes 

need "files" a character vector of filenames for the transcript-level abundances

```{r}

files <- list.files(path = ".", pattern = ".genes.results")
names(files) <- c("29_4cell_rep_1", "29_4cell_rep_2", "29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2", "29_blast_rep_3", "29_gast_rep_1", "29_gast_rep_2", "29_gast_rep_3", "29_larv_rep_1", "29_larv_rep_2", "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2", "33_gast_rep_3", "33_larv_rep_1", "33_larv_rep_2", "33_larv_rep_3", "eggs_rep_1", "eggs_rep_2", "eggs_rep_3")
txi.rsem <- tximport(files, type = "rsem")
head(txi.rsem$counts)
```
Yay these are integers1


Now I want to use DESeqDataSetFromTximport to make the DESeq object 

not able to do "intereaction effect" in this part of the design because eggs are only in the ambient treatment

```{r}
# Set up for complete combinatorial design 
Sample_info$Stage = factor(x = Sample_info$Stage,levels = c('fourcell','blastula', "gastrula", "larvae", "eggs"))
Sample_info$Temp = factor(x = Sample_info$Temp, levels = c("ambient", "high"))
ddsTxi_Comb <- DESeqDataSetFromTximport(txi.rsem,
                                   colData = Sample_info,
                                   design = ~ Stage + Temp)
```


Can pre-filter matrix to get rid of low-expressed genes 
```{r}
# then I need to filter out low count transcripts
# can't do pover a filtering because this is too new a version of R
# however I can make an educated guess for what I should do with filtering
# there are 27 samples
# 3 or 6 in each life stage
# lets go with 6 
# want at least 3 counts in six
# so 3*6 is 18
# DESeq manual says a good count to filter by is 10 so this seems reseaonable 
keep <- rowSums(counts(ddsTxi_Comb)) >= 18
length(keep[keep==FALSE])
# this removes 25 genes, not sure if this is a lot or not
ddsTxi_Comb_filt <- ddsTxi_Comb[keep,]

```
add interaction this way? from DESEq2 vingette
```{r}
ddsTxi_Comb_filt$group <- factor(paste0(ddsTxi_Comb_filt$Stage, ddsTxi_Comb_filt$Temp))
design(ddsTxi_Comb_filt) <- ~ group
dds <- DESeq(ddsTxi_Comb_filt)
resultsNames(dds)
print(dds$group)
```

All possible results combinations?

```{r}
# had to use F for 4 if starting the name of results
BA_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "blastulaambient"))
GA_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "gastrulaambient"))
LA_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "larvaeambient"))
EA_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "eggsambient"))
FCH_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "fourcellhigh"))
BH_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "blastulahigh"))
GH_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "gastrulahigh"))
LH_4CA <- results(dds, alpha=0.05, contrast=c("group", "fourcellambient", "larvaehigh"))

GA_BA <- results(dds, alpha=0.05, contrast=c("group", "blastulaambient", "gastrulaambient"))
LA_BA <- results(dds, alpha=0.05, contrast=c("group", "blastulaambient", "larvaeambient"))
EA_BA <- results(dds, alpha=0.05, contrast=c("group", "blastulaambient", "eggsambient"))
FCH_BA <- results(dds, alpha=0.05, contrast=c("group", "blastulaambient", "fourcellhigh"))
BH_BA <- results(dds, alpha=0.05, contrast=c("group", "blastulaambient", "blastulahigh"))
GH_BA <- results(dds, alpha=0.05, contrast=c("group", "blastulaambient", "gastrulahigh"))
LH_BA <- results(dds, alpha=0.05, contrast=c("group", "blastulaambient", "larvaehigh"))

LA_GA <- results(dds, alpha=0.05, contrast=c("group", "gastrulaambient", "larvaeambient"))
EA_GA <- results(dds, alpha=0.05, contrast=c("group", "gastrulaambient", "eggsambient"))
FCH_GA <- results(dds, alpha=0.05, contrast=c("group", "gastrulaambient", "eggsambient"))
BH_GA <- results(dds, alpha=0.05, contrast=c("group", "gastrulaambient", "blastulahigh"))
GH_GA <- results(dds, alpha=0.05, contrast=c("group", "gastrulaambient", "gastrulahigh"))
LH_GA <- results(dds, alpha=0.05, contrast=c("group", "gastrulaambient", "larvaehigh"))

EA_LA <- results(dds, alpha=0.05, contrast=c("group", "larvaeambient", "eggsambient"))
FCH_LA <- results(dds, alpha=0.05, contrast=c("group", "larvaeambient", "fourcellhigh"))
BH_LA <- results(dds, alpha=0.05, contrast=c("group", "larvaeambient", "blastulahigh"))
GH_LA <- results(dds, alpha=0.05, contrast=c("group", "larvaeambient", "gastrulahigh"))
LH_LA <- results(dds, alpha=0.05, contrast=c("group", "larvaeambient", "larvaehigh"))

FCH_EA <- results(dds, alpha=0.05, contrast=c("group", "eggsambient", "fourcellhigh"))
BH_EA <- results(dds, alpha=0.05, contrast=c("group", "eggsambient", "blastulahigh"))
GH_EA <- results(dds, alpha=0.05, contrast=c("group", "eggsambient", "gastrulahigh"))
LH_EA <- results(dds, alpha=0.05, contrast=c("group", "eggsambient", "larvaehigh"))

BH_4CH <- results(dds, alpha=0.05, contrast=c("group", "fourcellhigh", "blastulahigh"))
GH_4CH <- results(dds, alpha=0.05, contrast=c("group", "fourcellhigh", "gastrulahigh"))
LH_4CH <- results(dds, alpha=0.05, contrast=c("group", "fourcellhigh", "larvaehigh"))

GH_BH <- results(dds, alpha=0.05, contrast=c("group", "blastulahigh", "gastrulahigh"))
LH_BH <- results(dds, alpha=0.05, contrast=c("group", "blastulahigh", "larvaehigh"))

LH_GH <- results(dds, alpha=0.05, contrast=c("group", "gastrulahigh", "larvaehigh"))

```

Subset all of these to be only the significantly differentially expressed 
```{r}
sig_BA_4CA <- subset(BA_4CA, padj<0.05)
sig_GA_4CA <- subset(GA_4CA, padj<0.05)
sig_LA_4CA <- subset(LA_4CA, padj<0.05)
sig_EA_4CA <- subset(EA_4CA, padj<0.05)
sig_4CH_4CA <- subset(FCH_4CA, padj<0.05)
sig_BH_4CA <- subset(BH_4CA, padj<0.05)
sig_GH_4CA <- subset(GH_4CA, padj<0.05)
sig_LH_4CA <- subset(LH_4CA, padj<0.05)

sig_GA_BA <- subset(GA_BA, padj<0.05)
sig_LA_BA <- subset(LA_BA, padj<0.05)
sig_EA_BA <- subset(EA_BA, padj<0.05)
sig_4CH_BA <- subset(FCH_BA, padj<0.05)
sig_BH_BA <- subset(BH_BA, padj<0.05)
sig_GH_BA <- subset(GH_BA, padj<0.05)
sig_LH_BA <- subset(LH_BA, padj<0.05)

sig_LA_GA <- subset(LA_GA, padj<0.05)
sig_EA_GA <- subset(EA_GA, padj<0.05)
sig_4CH_GA <- subset(FCH_GA, padj<0.05)
sig_BH_GA <- subset(BH_GA, padj<0.05)
sig_GH_GA <- subset(GH_GA, padj<0.05)
sig_LH_GA <- subset(LH_GA, padj<0.05)

sig_EA_LA <- subset(EA_LA, padj<0.05)
sig_4CH_LA <- subset(FCH_LA, padj<0.05)
sig_BH_LA <- subset(BH_LA, padj<0.05)
sig_GH_LA <- subset(GH_LA, padj<0.05)
sig_LH_LA <- subset(LH_LA, padj<0.05)

sig_4CH_EA <- subset(FCH_EA, padj<0.05)
sig_BH_EA <- subset(BH_EA, padj<0.05)
sig_GH_EA <- subset(GH_EA, padj<0.05)
sig_LH_EA <- subset(LH_EA, padj<0.05)

sig_BH_4CH <- subset(BH_4CH, padj<0.05)
sig_GH_4CH <- subset(GH_4CH, padj<0.05)
sig_LH_4CH <- subset(LH_4CH, padj<0.05)

sig_GH_BH <- subset(GH_BH, padj<0.05)
sig_LH_BH <- subset(LH_BH, padj<0.05)

sig_LH_GH <- subset(LH_GH, padj<0.05)
```

Create just Stage and just Temp Models

Stage

```{r}
ddsTxi_Stage <- DESeqDataSetFromTximport(txi.rsem,
                                   colData = Sample_info,
                                   design = ~ Stage)
```

```{r}
# then I need to filter out low count transcripts
# can't do pover a filtering because this is too new a version of R
# however I can make an educated guess for what I should do with filtering
# there are 27 samples
# 3 or 6 in each life stage
# lets go with 6 
# want at least 3 counts in six
# so 3*6 is 18
# DESeq manual says a good count to filter by is 10 so this seems reseaonable 
keep <- rowSums(counts(ddsTxi_Stage)) >= 18
length(keep[keep==FALSE])
# this removes 25 genes, not sure if this is a lot or not
ddsTxi_Stage_filt <- ddsTxi_Stage[keep,]

```

```{r}
dds_Stage <- DESeq(ddsTxi_Stage_filt)

resultsNames(dds_Stage)
```
Get results for all stage comparisons 

```{r}
Blastula_4cell <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "blastula", "fourcell"))
Gastrula_4cell <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "gastrula", "fourcell"))
Larvae_4cell <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "larvae", "fourcell"))
Eggs_4cell <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "eggs", "fourcell"))

Blastula_Eggs <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "blastula", "eggs"))
Gastrula_Eggs <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "gastrula", "eggs"))
Larvae_Eggs <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "larvae", "eggs"))

Blastula_Larvae <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "blastula", "larvae"))
Gastrula_Larvae <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "gastrula", "larvae"))

Blastula_Gastrula <- results(dds_Stage, alpha=0.05, contrast=c("Stage", "blastula", "gastrula"))

summary(Blastula_Gastrula)
```

Subset those to only DEGs

```{r}
sig_Blastula_4cell <- subset(Blastula_4cell, padj<0.05)
sig_Gastrula_4cell <- subset(Gastrula_4cell, padj<0.05)
sig_Larvae_4cell <- subset(Larvae_4cell, padj<0.05)
sig_Eggs_4cell <- subset(Eggs_4cell, padj<0.05)

sig_Blastula_Eggs <- subset(Blastula_Eggs, padj<0.05)
sig_Gastrula_Eggs <- subset(Blastula_Eggs, padj<0.05)
sig_Larvae_Eggs <- subset(Blastula_Eggs, padj<0.05)

sig_Blastula_Larvae <- subset(Blastula_Larvae, padj<0.05)
sig_Gastrula_Larvae <- subset(Gastrula_Larvae, padj<0.05)

sig_Blastula_Gastrula <- subset(Gastrula_Larvae, padj<0.05)
```

Temp only 
```{r}
ddsTxi_Temp <- DESeqDataSetFromTximport(txi.rsem,
                                   colData = Sample_info,
                                   design = ~ Temp)
```
```{r}
# then I need to filter out low count transcripts
# can't do pover a filtering because this is too new a version of R
# however I can make an educated guess for what I should do with filtering
# there are 27 samples
# 3 or 6 in each life stage
# lets go with 6 
# want at least 3 counts in six
# so 3*10 is 30
# DESeq manual says a good count to filter by is 10 so this seems reseaonable 
keep <- rowSums(counts(ddsTxi_Temp)) >=30
length(keep[keep==FALSE])
# this removes 114 genes, not sure if this is a lot or not
ddsTxi_Temp_filt <- ddsTxi_Temp[keep,]

```

```{r}
dds_Temp <- DESeq(ddsTxi_Temp_filt)

resultsNames(dds_Temp)
```
Only one comparison here

```{r}
temp_amb_high <- results(dds_Temp, alpha=0.05, contrast=c("Temp", "high", "ambient"))
summary(temp_amb_high)
```
There are no significantly differentially expressed genes for temperature.... 

Overall....

There are some stage-specific DEGs between the treatments, but not all

```{r}
summary(GH_GA)
summary(BH_BA)
summary(LH_LA)
summary(FCH_4CA)
```

```{r}
# write tables of the stage specific DEGs for temp

write.table(sig_GH_GA,"sig_GH_GA.txt",quote=FALSE,col.names=TRUE,row.names=TRUE,sep="\t")
write.table(sig_BH_BA,"sig_BH_BA.txt",quote=FALSE,col.names=TRUE,row.names=TRUE,sep="\t")
write.table(sig_LH_LA,"sig_LH_LA.txt",quote=FALSE,col.names=TRUE,row.names=TRUE,sep="\t")
```

Exploritory Data Visualizations

# general PCAs of the data
# using varience stablizing transformations 
```{r}
ddsvst <- vst(dds, blind = FALSE)
plotPCA(ddsvst, intgroup=c("Stage"))
plotPCA(ddsvst, intgroup=c("Temp"))
```

High Quality PCA of all genes, separated by stage

```{r}
pcaData <- plotPCA(ddsvst, intgroup=c("Stage"), returnData=TRUE)
percentVar <- round(100 * attr(pcaData, "percentVar"))
ggplot(pcaData, aes(PC1, PC2, color=Stage)) +
  geom_point(size=3) +
  coord_fixed() + ylab("PC2: 14% Variance Explained") +
  xlab("PC1: 80% Variance Explained") +
  theme_linedraw() + scale_color_manual(values = c("fourcell" = "#ED6A5A", "blastula" = "#0D5C63", "gastrula" = "#FED766", "larvae" = "#A03E99", "eggs" = "#9DD9D2"))


```

Can I plot the 3rd and 4th PCs? Of all genes

```{r}
# different way of doing PCA
mat <- assay(ddsvst)
# need to transpose to tidy format for this PCA format
tmat <- as.data.frame(t(mat))
PCEL <- prcomp(tmat)
#loadings
head(PCEL$rotation)
# scores
head(PCEL$x)

scores = as.data.frame(PCEL$x)

# plot of observations
ggplot(data = scores, aes(x = PC3, y = PC4, label = rownames(scores), color = )) +
  geom_hline(yintercept = 0, colour = "gray65") +
  geom_vline(xintercept = 0, colour = "gray65") +
  geom_text(colour = "tomato", alpha = 0.8, size = 4)
```
Does not look like there is clustering by temp treatment 





Sample to Sample distance matrix using all genes. Code form Erin Chille

The darker blue the less "distance" between the two samples
```{r}
gsampleDists <- dist(t(assay(ddsvst))) #calculate distance matix
gsampleDistMatrix <- as.matrix(gsampleDists) #distance matrix
rownames(gsampleDistMatrix) <- colnames(ddsvst) #assign row names
colnames(gsampleDistMatrix) <- NULL #assign col names
colors <- colorRampPalette( rev(brewer.pal(9, "Blues")) )(255) #assign colors
pheatmap(gsampleDistMatrix, #plot matrix
         clustering_distance_rows=gsampleDists, #cluster rows
         clustering_distance_cols=gsampleDists, #cluster columns
         col=colors) #set colors


```

In the clustering it doesn't look like the temp treatments within stages cluster... 

And it shows no differentiation in overall expression from egg to 4cell 


More specific PCAs using differentially expressed genes only. 

Stage

```{r}
# first have to combine all the significant lists
stage_combine <- rbind(sig_Blastula_4cell, sig_Gastrula_4cell, sig_Larvae_4cell, sig_Eggs_4cell, sig_Blastula_Eggs, sig_Gastrula_Eggs, sig_Larvae_Eggs, sig_Blastula_Larvae, sig_Gastrula_Larvae, sig_Blastula_Gastrula)

stage_list <- ddsTxi_Stage[which(rownames(ddsTxi_Stage) %in% rownames(stage_combine)),] 


stage_vst <- vst(stage_list)
plotPCA(stage_vst, intgroup=c("Stage"))
```


Heat map of all genes

```{r}
select <- order(rowMeans(counts(dds,normalized=TRUE)),
                decreasing=TRUE)[1:20]
df <- as.data.frame(colData(dds)[,c("Stage","Temp")])
pheatmap(assay(ddsvst)[select,], cluster_rows=TRUE, show_rownames=FALSE,
         cluster_cols=TRUE, annotation_col=df)


```



```{r}
# sig.num <- sum(combine$padj <0.1, na.rm=T) 
# topVarGenes <- head(order(rowVars(assay(combine_rlog)),decreasing=TRUE),sig.num) #sort by decreasing sig
# mat <- assay(combine_rlog)[ topVarGenes, ] #make an expression object

# mat <- mat - rowMeans(mat) #difference in expression compared to average across all samples
# col.order <- c("29_4cell_rep_1", "29_4cell_rep_2", "29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2", "29_blast_rep_3", "29_gast_rep_1", "29_gast_rep_2", "29_gast_rep_3", "29_larv_rep_1", "29_larv_rep_2", "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2", "33_gast_rep_3", "33_larv_rep_1", "33_larv_rep_2", "33_larv_rep_3", "eggs_rep_1", "eggs_rep_2", "eggs_rep_3")
# mat <- mat[,col.order]
# df1 <- as.data.frame(colData(combine_rlog)[c("effluent", "pCO2")]) #make dataframe for column naming 


select <- order(rowMeans(counts(dds,normalized=TRUE)),
                decreasing=TRUE)[1:50]

matri <- assay(ddsvst)[select,]
col.order <- c("29_4cell_rep_1", "29_4cell_rep_2", "29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2", "29_blast_rep_3", "29_gast_rep_1", "29_gast_rep_2", "29_gast_rep_3", "29_larv_rep_1", "29_larv_rep_2", "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2", "33_gast_rep_3", "33_larv_rep_1", "33_larv_rep_2", "33_larv_rep_3", "eggs_rep_1", "eggs_rep_2", "eggs_rep_3")
matri <- matri[,col.order]



df <- as.data.frame(colData(dds)[,c("Stage","Temp")])

colfunc <- colorRampPalette(c("steelblue3", "white", "darkorange1")) #make function for the color gradient 
ann_colors <- list(Temp = c(ambient="skyblue1", high="tomato3"), Stage = c(fourcell= "#ED6A5A", blastula="#0D5C63",gastrula= "#FED766",larvae= "#A03E99",eggs= "#9DD9D2"))
# breakss <- c(-2, -1.9, -1.8, -1.7, -1.6, -1.5, -1.4, -1.3, -1.2, -1.1, -1, -.9, -.8, -.7, -.6, -.5, -.4, -.3, -.2, -.1, 0, .1, .2, .3, .4, .5, .6, .7, .8, .9, 1, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 2) 
pheatmap(matri, annotation_col=df, clustering_method = "average", 
         clustering_distance_rows="euclidean", show_rownames =FALSE, cluster_cols=TRUE,
         show_colnames =F, annotation_colors=ann_colors, color = colfunc(50))

```




heatmap of stage DEGs

```{r}
# using the combined list 
rlog_stage <- rlog(stage_list, blind = FALSE)
sig.num <- sum(stage_combine$padj <0.00000000005, na.rm=T) 
topVarGenes <- head(order(rowVars(assay(rlog_stage)),decreasing=TRUE),sig.num) #sort by decreasing sig
mat <- assay(rlog_stage)[ topVarGenes, ] #make an expression object

mat <- mat - rowMeans(mat) #difference in expression compared to average across all samples
col.order <- c("29_4cell_rep_1", "29_4cell_rep_2", "29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2", "29_blast_rep_3", "29_gast_rep_1", "29_gast_rep_2", "29_gast_rep_3", "29_larv_rep_1", "29_larv_rep_2", "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2", "33_gast_rep_3", "33_larv_rep_1", "33_larv_rep_2", "33_larv_rep_3", "eggs_rep_1", "eggs_rep_2", "eggs_rep_3")
mat <- mat[,col.order]
df1 <- as.data.frame(colData(rlog_stage)[c("Stage", "Temp")]) #make dataframe for column naming 

colfunc <- colorRampPalette(c("steelblue3", "white", "darkorange1")) #make function for the color gradient 
ann_colors <- list(Temp = c(ambient="skyblue1", high="tomato3"), Stage = c(fourcell= "#ED6A5A", blastula="#0D5C63",gastrula= "#FED766",larvae= "#A03E99",eggs= "#9DD9D2"))
# breakss <- c(-2, -1.9, -1.8, -1.7, -1.6, -1.5, -1.4, -1.3, -1.2, -1.1, -1, -.9, -.8, -.7, -.6, -.5, -.4, -.3, -.2, -.1, 0, .1, .2, .3, .4, .5, .6, .7, .8, .9, 1, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 2) 
pheatmap(mat, annotation_col=df1, clustering_method = "average", 
         clustering_distance_rows="euclidean", show_rownames =FALSE, cluster_cols=TRUE ,
         show_colnames =F, annotation_colors=ann_colors, color = colfunc(50))



```

```{r}
temp_comb <- rbind(sig_GH_GA, sig_BH_BA, sig_LH_LA)

temp_list <- ddsTxi_Stage[which(rownames(ddsTxi_Stage) %in% rownames(temp_comb)),] 

rlog_templist <- rlog(temp_list, blind = FALSE)

```







```{r}

sig.num <- sum(temp_comb$padj <0.05, na.rm=T) 
topVarGenes <- head(order(rowVars(assay(rlog_templist)),decreasing=TRUE),sig.num) #sort by decreasing sig
mat <- assay(rlog_stage)[ topVarGenes, ] #make an expression object

mat <- mat - rowMeans(mat) #difference in expression compared to average across all samples
col.order <- c("eggs_rep_1", "eggs_rep_2", "eggs_rep_3",  "29_4cell_rep_1", "29_4cell_rep_2", "29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2", "29_blast_rep_3", "29_gast_rep_1", "29_gast_rep_2", "29_gast_rep_3", "29_larv_rep_1", "29_larv_rep_2", "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2", "33_gast_rep_3", "33_larv_rep_1", "33_larv_rep_2", "33_larv_rep_3")
mat <- mat[,col.order]
df1 <- as.data.frame(colData(rlog_templist)[c("Stage", "Temp")]) #make dataframe for column naming 

colfunc <- colorRampPalette(c("steelblue3", "white", "darkorange1")) #make function for the color gradient 
ann_colors <- list(Temp = c(ambient="skyblue1", high="tomato3"), Stage = c(fourcell= "#ED6A5A", blastula="#0D5C63",gastrula= "#FED766",larvae= "#A03E99",eggs= "#9DD9D2"))
# breakss <- c(-2, -1.9, -1.8, -1.7, -1.6, -1.5, -1.4, -1.3, -1.2, -1.1, -1, -.9, -.8, -.7, -.6, -.5, -.4, -.3, -.2, -.1, 0, .1, .2, .3, .4, .5, .6, .7, .8, .9, 1, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7, 1.8, 1.9, 2) 
pheatmap(mat, annotation_col=df1, clustering_method = "average", 
         clustering_distance_rows="euclidean", show_rownames =FALSE, cluster_cols=FALSE ,
         show_colnames =F, annotation_colors=ann_colors, color = colfunc(50))




```