---
title: "GRN-Count-Vis"
author: "Maggie Schedl"
date: "6/30/2020"
output: github_document
---

```{r}
library(DESeq2)
library(ggplot2)
library(pheatmap)
library(RColorBrewer)
library(reshape2)
library(rsconnect)
library(gplots)
library(ashr)
library(dplyr)
library(tidyr)
library(dlookr)
library(tximport)
library(naniar)
library(tidyverse)
library(dplyr)
library(stringr)
library(ggpubr)
```

First I want the the Trinity transcript names of the GRN genes

Read in orthogroups file and do same file manipulations as in annotation script
```{r}
orthologs <- read.delim("LORF_EL90.pep__v__SPU_peptide.tsv", sep = "\t")
head(orthologs)
EL.orthologs <- orthologs %>% 
  select(1,2)
head(EL.orthologs)
# collapse df in to one column multiple rows instead of one row multiple columns
tr.orthogroups <- separate_rows(EL.orthologs, "LORF_EL90.pep", sep = ",", convert = FALSE)
head(tr.orthogroups)
# remove extra characters of .p#s
tr.orthogroups <-mapply(gsub, pattern = ".p1", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# I don't know why this always removes the column names but it does
# remove .p2
tr.orthogroups <-mapply(gsub, pattern = ".p2", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# remove .p3
tr.orthogroups <-mapply(gsub, pattern = ".p3", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# remove .p4
tr.orthogroups <-mapply(gsub, pattern = ".p4", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# remove .p5
tr.orthogroups <-mapply(gsub, pattern = ".p5", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# add column names
colnames(tr.orthogroups) <- c("Orthogroup", "transcript_id")
head(tr.orthogroups)


# remove the isoform notations from this file, because the counts file is only by gene. Its ok to collapse by uniquing the transcript ID because they should be in the same orthogroup 
# I can use . as any character! 
tr.orthogroups.genes <-mapply(gsub, pattern = "_i.", replacement = "", tr.orthogroups)
tr.orthogroups.genes <- as.data.frame(tr.orthogroups.genes)
# there might be random spaces in the file??
tr.orthogroups.genes <-mapply(gsub, pattern = " ", replacement = "", tr.orthogroups.genes)
tr.orthogroups.genes <- as.data.frame(tr.orthogroups.genes)
colnames(tr.orthogroups.genes) <- c("Orthogroup", "transcript_id")
tr.orthogroups.genes.d <- unique(tr.orthogroups.genes)

tr.orthogroups.genes[1:100,]
tr.orthogroups.genes.d[1:100,]
```

Bring in the counts file 
this is from the full model of DESeq2 that was based on the comparison between stages
```{r}
normalized_counts_stage <- read.delim("normalized_counts_stage.txt", sep = "\t")
head(normalized_counts_stage)

colnames(normalized_counts_stage) <- c("transcript_id","29_4cell_rep_1" ,"29_4cell_rep_2" ,"29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2" ,"29_blast_rep_3", "29_gast_rep_1",  "29_gast_rep_2",  "29_gast_rep_3", "29_larv_rep_1",  "29_larv_rep_2",  "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2",  "33_gast_rep_3",  "33_larv_rep_1",  "33_larv_rep_2",  "33_larv_rep_3", "eggs_rep_1" , "eggs_rep_2",  "eggs_rep_3"  )
head(normalized_counts_stage)
```

VST transformed counts?
```{r}
vst_counts_stage <- read.delim("vst_stage_counts.txt", sep = "\t", header = TRUE)
head(vst_counts_stage)
vst_counts_stage$transcript_id <- rownames(vst_counts_stage)
colnames(vst_counts_stage) <- c("29_4cell_rep_1" ,"29_4cell_rep_2" ,"29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2" ,"29_blast_rep_3", "29_gast_rep_1",  "29_gast_rep_2",  "29_gast_rep_3", "29_larv_rep_1",  "29_larv_rep_2",  "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2",  "33_gast_rep_3",  "33_larv_rep_1",  "33_larv_rep_2",  "33_larv_rep_3", "eggs_rep_1" , "eggs_rep_2",  "eggs_rep_3", "transcript_id"  )
rownames(vst_counts_stage) <- NULL
head(vst_counts_stage)

```

rlog transformed counts?

```{r}
rlog_counts_stage <- read.delim("rlog_stage_counts.txt", sep = "\t", header = TRUE)
head(rlog_counts_stage)
rlog_counts_stage$transcript_id <- rownames(rlog_counts_stage)
colnames(rlog_counts_stage) <- c("29_4cell_rep_1" ,"29_4cell_rep_2" ,"29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2" ,"29_blast_rep_3", "29_gast_rep_1",  "29_gast_rep_2",  "29_gast_rep_3", "29_larv_rep_1",  "29_larv_rep_2",  "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2",  "33_gast_rep_3",  "33_larv_rep_1",  "33_larv_rep_2",  "33_larv_rep_3", "eggs_rep_1" , "eggs_rep_2",  "eggs_rep_3", "transcript_id"  )
rownames(rlog_counts_stage) <- NULL
head(rlog_counts_stage)

```


Want to merge the counts file and the orthogroup file

```{r}
counts_stage_orthogroups <- merge(normalized_counts_stage, tr.orthogroups.genes.d, by= "transcript_id", sort = TRUE )
head(counts_stage_orthogroups)
dim(normalized_counts_stage) # 13296
dim(counts_stage_orthogroups) # 13226

# not all the ones in the original orthogroups file are present because lowly expressed genes were filtered out in making the normalized count matrix
```
combine vst counts with orthogroups
```{r}
vst_counts_stage_orthogroups <- merge(vst_counts_stage, tr.orthogroups.genes.d, by= "transcript_id", sort = TRUE )
head(vst_counts_stage_orthogroups)

```

combine rlog counts with orthogroups
```{r}
rlog_counts_stage_orthogroups <- merge(rlog_counts_stage, tr.orthogroups.genes.d, by= "transcript_id", sort = TRUE )
head(rlog_counts_stage_orthogroups)

```



Now to limit the anotation file that has orthogroups to the GRN genes to find the orthogroups of the GRN genes 

read in annotation file limited by orthogroups
```{r}
annot.ortho.SPU <- read.delim("annot.ortho.SPU.txt", sep = "\t")
head(annot.ortho.SPU)

```

Read in GRN file and make vector of gene names 
```{r}
GRN <- read.csv("journal.pbio.1002391.s013.GRN.csv", header = TRUE)
head(GRN)
dim(GRN) #126

GRN.genes <- GRN$Gene.Name

```


subset
```{r}
annot.otho.GRNs <- subset(annot.ortho.SPU, common_name %in% GRN.genes)
print(annot.otho.GRNs) 


```

Subset to be just othogroup and common name and merge with counts file

```{r}
ortho.GRN <- annot.otho.GRNs[,c(2,4)]

dim(ortho.GRN) #69

EL.ortho.GRN.counts <- merge(counts_stage_orthogroups, ortho.GRN, by= "Orthogroup", sort = TRUE)
EL.ortho.GRN.counts
dim(EL.ortho.GRN.counts) #67

# so that means at least 2 of the GRN genes may have had too low counts to measure
# but hard to tell because multiple of the transcripts will match 
```

merge with vst counts file

```{r}
vst.EL.ortho.GRN.counts <- merge(vst_counts_stage_orthogroups, ortho.GRN, by= "Orthogroup", sort = TRUE)
vst.EL.ortho.GRN.counts
dim(vst.EL.ortho.GRN.counts) #67

```

merge with rlog counts file

```{r}
rlog.EL.ortho.GRN.counts <- merge(rlog_counts_stage_orthogroups, ortho.GRN, by= "Orthogroup", sort = TRUE)
rlog.EL.ortho.GRN.counts
dim(rlog.EL.ortho.GRN.counts) #67

```

play with vst gene count dataset to get to tidy format 
```{r}
t.vst <- as.data.frame(t(vst.EL.ortho.GRN.counts))
head(t.vst)
colnames(t.vst) <- vst.EL.ortho.GRN.counts$common_name
t.vst
Sample_info <- read.csv("el-treatment-data.csv", header=TRUE, sep=",")
t.vst <- t.vst[-c(1,2,30),]

t.vst.samp <- cbind(Sample_info, t.vst)
t.vst.samp
colnames(t.vst)
colnames(t.vst.samp) <- c("Sample", "Stage", "Temp", "p58_a",  "Ets1_2" , "FoxQ2_1" ,  "Erg"     ,  "Ttrspn_19", "Cara7LA"  , "Not"   ,    "Not_2"   ,    "Eve"    ,   "Blimp1"  ,  "RhoA"  ,    "Gcm"  ,     "Sm37", "Otx"   ,    "Pks2"   ,   "Tbr"   ,    "Tel"    ,   "Sp5"  ,     "Myc"  ,     "Cycpln"    ,"IrxA"   ,   "Wnt5" ,     "Emx"    ,   "Wnt1"  ,    "Wnt16"  ,   "Sm29" ,"p58_b"  ,   "Wnt4"    ,  "Delta"  ,   "FoxG"     , "GataC"    , "ScratchX" , "Msxl"   ,   "FoxB"   ,   "Endo16"  ,  "E2f3"  ,    "Nodal" ,    "Chordin" ,  "Wnt8", "Tgif"   ,   "FoxA"  ,    "Alx1"    ,  "Vegf3"  ,   "Sip1"   ,   "Pitx2"   ,  "Nk1"    ,   "SoxE"   ,   "Lox"     ,  "Wnt10"  ,   "FoxC"   ,   "Sm50"    ,  "Msp130", "HesC"   ,   "C-lectin" , "SoxC"    ,  "Shr2"  ,    "FoxY"  ,    "Ese"   ,    "Gatae"   ,  "Gsc"   ,    "Lefty"    , "Six3"  ,    "Bra"  ,     "Hox7"  ,    "Hh", "P19"   ,    "FoxF")
t.vst.samp
# to order the stages in chronological order
t.vst.samp$Stage <- factor(t.vst.samp$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)

for(i in 4:ncol(t.vst.samp)) {
t.vst.samp[,i] <- as.numeric(as.character(t.vst.samp[,i]))
}

t.vst.samp$p58_a <- as.numeric(as.character(t.vst.samp$p58_a))


write.table(t.vst.samp,"t.vst.samp.txt",quote=FALSE,col.names=TRUE,row.names=TRUE,sep="\t")
```


"p58_a",  "Ets1_2" , "FoxQ2_1" ,  "Erg"     ,  "Ttrspn_19", "Cara7LA"  , "Not"   ,    "Not_2"   ,    "Eve"    ,   "Blimp1"  ,  "RhoA"  ,    "Gcm"  ,     "Sm37", "Otx"   ,    "Pks2"   ,   "Tbr"   ,    "Tel"    ,   "Sp5"  ,     "Myc"  ,     "Cycpln"    ,"IrxA"   ,   "Wnt5" ,     "Emx"    ,   "Wnt1"  ,    "Wnt16"  ,   "Sm29" ,"p58_b"  ,   "Wnt4"    ,  "Delta"  ,   "FoxG"     , "GataC"    , "ScratchX" , "Msxl"   ,   "FoxB"   ,   "Endo16"  ,  "E2f3"  ,    "Nodal" ,    "Chordin" ,  "Wnt8", "Tgif"   ,   "FoxA"  ,    "Alx1"    ,  "Vegf3"  ,   "Sip1"   ,   "Pitx2"   ,  "Nk1"    ,   "SoxE"   ,   "Lox"     ,  "Wnt10"  ,   "FoxC"   ,   "Sm50"    ,  "Msp130", "HesC"   ,   "C-lectin" , "SoxC"    ,  "Shr2"  ,    "FoxY"  ,    "Ese"   ,    "Gatae"   ,  "Gsc"   ,    "Lefty"    , "Six3"  ,    "Bra"  ,     "Hox7"  ,    "Hh", "P19"   ,    "FoxF"

```{r}

vst_p58_a_point <- ggplot(t.vst.samp, aes(x=Stage, y=p58_a, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_p58_a")
vst_p58_a_point

vst_Erg_point <- ggplot(t.vst.samp, aes(x=Stage, y=Erg, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Erg")
vst_Erg_point

vst_Ets1_2_point <- ggplot(t.vst.samp, aes(x=Stage, y=Ets1_2, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Ets1_2")
vst_Ets1_2_point

vst_Ttrspn_19_point <- ggplot(t.vst.samp, aes(x=Stage, y=Ttrspn_19, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Ttrspn_19")
vst_Ttrspn_19_point

vst_Cara7LA_point <- ggplot(t.vst.samp, aes(x=Stage, y=Cara7LA, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Cara7LA")
vst_Cara7LA_point

vst_Not_point <- ggplot(t.vst.samp, aes(x=Stage, y=Not, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Not")
vst_Not_point

vst_Not_2_point <- ggplot(t.vst.samp, aes(x=Stage, y=Not_2, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Not_2")
vst_Not_2_point

vst_Eve_point <- ggplot(t.vst.samp, aes(x=Stage, y=Eve, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Eve")
vst_Eve_point

vst_Blimp1_point <- ggplot(t.vst.samp, aes(x=Stage, y=Blimp1, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_Blimp1")
vst_Blimp1_point

vst_Rhoy_point <- ggplot(t.vst.samp, aes(x=Stage, y=RhoA, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_RhoA")
vst_Rhoy_point
```

```{r}
# scale? so they all can be plot on one??
t.vst
t.vst.scale <- t.vst[-c(1:3),]
t.vst.scale
```

code from Devon DeRaad
```{r}
# separate out row number
t.vst<-read.table("t.vst.samp.txt")

tidy.df<-data.frame()
for (i in 4:ncol(t.vst)){
  chunk<-t.vst[,c(2,i)] # take dataframe and keep all rows and column 2 and column i and saves a s "chunk"
  chunk<-as.data.frame(cbind(chunk,rep(colnames(t.vst)[i]))) # bind chunk dataframe and add other column on it that is the name of the gene 
  colnames(chunk)[2:3]<-c("expression","gene")
  tidy.df<-as.data.frame(rbind(tidy.df,chunk))
}

plot_list <- list()
for (i in levels(as.factor(tidy.df$gene))) {
  #subset df to only gene of interest
  t.gene_samp_info<-tidy.df[tidy.df$gene == i,]
  #reorder stage
  t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage, levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
  #ggplot
  plot_list[[i]]<-ggplot(t.gene_samp_info, aes(x=Stage, y=expression, group =Stage)) + 
    geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle(i)

}
```




```{r}
tidy.df$Stage <- factor(tidy.df$Stage, levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
ggplot(tidy.df, aes(x=Stage, y=expression, group =Stage, color=gene)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=gene, color=gene)) + theme_linedraw()
```

split these up by sig degs 
```{r}
fourcell_egg_GRN_counts$common_name
# need to somehow subset tidy.df to only these 

FC_E_DEG_counts <- filter(tidy.df, gene %in% c("p58_a"  , "FoxQ2_1", "Eve"  ,   "Sm37",    "Tel"  ,   "Wnt1" ,   "Nodal" ,  "Wnt8" ,   "HesC"  ,  "Lefty"  , "Hox7" ,"P19"    , "FoxF"))
FC_E_DEG_counts
# relevel
FC_E_DEG_counts$Stage <- factor(FC_E_DEG_counts$Stage, levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
ggplot(FC_E_DEG_counts, aes(x=Stage, y=expression, group =Stage, color=gene)) + 
  geom_line() + stat_summary(fun=mean, geom="line", aes(group=gene, color=gene)) + theme_linedraw() + ylab("VST transformed normalized counts")
 
```

```{r}
Blastula_4cell_GRN_counts$common_name
B_FC_DEG_counts <- filter(tidy.df, gene %in% c( "p58_a"  ,   "Ets1/2" ,   "FoxQ2_1" ,  "Erg"     ,  "Ttrspn_19" ,"Cara7LA" ,  "Not" ,      "Not_2"    ,   "Eve","Blimp1"  ,  "Gcm"   ,    "Sm37"    ,  "Otx"    ,   "Pks2"   ,   "Tbr"    ,   "Tel"  ,     "Sp5"    ,   "Myc" , "Cycpln"  ,  "IrxA"  ,    "Emx"   ,    "Wnt1"   ,   "Wnt16"   ,  "Sm29" ,     "p58_b"   ,  "Delta"   ,  "FoxG", "GataC"  ,   "ScratchX" , "Msxl"   ,   "FoxB"  ,    "Endo16"   , "E2f3" ,     "Nodal"   ,  "Chordin"  , "Wnt8" ,"Sm50"   ,   "Msp130"  ,  "HesC"  ,    "C-lectin",  "SoxC" ,     "Shr2"   ,   "FoxY"    ,  "Ese"    ,   "Gatae" ,"Gsc"  ,     "Lefty"    , "Six3"   ,   "Bra"     ,  "Hox7"    ,  "Hh"      ,  "P19"      , "FoxF"))
# relevel
B_FC_DEG_counts$Stage <- factor(B_FC_DEG_counts$Stage, levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
ggplot(B_FC_DEG_counts, aes(x=Stage, y=expression, group =Stage, color=gene)) + 
  geom_line() + stat_summary(fun=mean, geom="line", aes(group=gene, color=gene)) + theme_linedraw() + ylab("VST transformed normalized counts")

```

```{r}
Blastula_Gastrula_GRN_counts$common_name
B_G_DEG_counts <- filter(tidy.df, gene %in% c("Ets1_2"  ,  "FoxQ2_1"  , "Erg"    ,   "Ttrspn_19" ,"Cara7LA",   "Not"   ,    "Not_2"   ,    "Eve"    ,   "Blimp1"   , "RhoA"   ,   "Gcm"    ,   "Sm37", "Otx"   ,    "Pks2"  ,    "Tbr"   ,    "Tel",       "Myc"  ,     "Wnt16"  ,   "Sm29"  ,    "GataC"  ,   "ScratchX" , "FoxB",      "Endo16"  ,  "Chordin" ,"Alx1"    ,  "Vegf3"   ,  "Pitx2"   ,  "SoxE"   ,   "Lox"   ,    "Wnt10"    , "FoxC"   ,   "Sm50"    ,  "Msp130"   , "Shr2" ,     "Gsc"   ,    "Hh","FoxF"))
B_G_DEG_counts$Stage <- factor(B_G_DEG_counts$Stage, levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
ggplot(B_G_DEG_counts, aes(x=Stage, y=expression, group =Stage, color=gene)) + 
  geom_line() + stat_summary(fun=mean, geom="line", aes(group=gene, color=gene)) + theme_linedraw() + ylab("VST transformed normalized counts")
  
B_G_DEG_counts2 <- filter(tidy.df, Stage %in% c("blastula", "gastrula"))
expression <- B_G_DEG_counts2
scaled_express <- scale(expression)

B_G_DEG_counts2.scaled <- cbind(B_G_DEG_counts2, scaled_express)
B_G_DEG_counts2.scaled


ggplot(B_G_DEG_counts2, aes(x=Stage, y=expression, group =Stage, color=gene)) + 
  geom_line() + stat_summary(fun=mean, geom="line", aes(group=gene, color=gene)) + theme_linedraw() + ylab("VST transformed normalized counts")
```

```{r}
Gastrula_Larvae_GRN_counts$common_name
G_L_DEG_counts <- filter(tidy.df, gene %in% c("Ets1_2"  ,  "FoxQ2_1" ,  "Erg"  ,     "Ttrspn_19" ,"Cara7LA" ,  "Not" ,      "Not_2"    ,   "Eve"     ,  "Blimp1"  ,  "RhoA"    ,  "Gcm"     ,  "Sm37", "Otx"     ,  "Pks2"    ,  "Tbr"    ,   "Tel"   ,    "Myc"  ,    "Wnt16"   ,  "Sm29"   ,   "GataC"   ,  "ScratchX"  ,"FoxB"  ,   "Endo16"  ,  "Chordin", "Alx1"   ,   "Vegf3"  ,   "Pitx2"    , "SoxE"     , "Lox"   ,    "Wnt10"   ,  "FoxC"  ,    "Sm50"     , "Msp130" ,   "Shr2"  ,    "Gsc"   ,    "Hh", "FoxF"))
G_L_DEG_counts$Stage <- factor(G_L_DEG_counts$Stage, levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
ggplot(G_L_DEG_counts, aes(x=Stage, y=expression, group =Stage, color=gene)) + 
  geom_line() + stat_summary(fun=mean, geom="line", aes(group=gene, color=gene)) + theme_linedraw() + ylab("VST transformed normalized counts")
```




plot p58-a using vst counts

```{r}
# separate out row number
gene<- vst.EL.ortho.GRN.counts[1,]
# OG0000751	TRINITY_DN30130_c0_g1
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
vst_p58_a_point <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("vst_p58_a")
vst_p58_a_point

```
Looks really similar, which is good, but the scale is so much nicer

rlog transformed

```{r}
# separate out row number
gene<- rlog.EL.ortho.GRN.counts[1,]
# OG0000751	TRINITY_DN30130_c0_g1
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
rlog_p58_a_point <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, group =Stage)) + 
  geom_point() + stat_summary(fun=mean, geom="line", aes(group=1)) + theme_linedraw() + ggtitle("rlog_p58_a")
rlog_p58_a_point

```


Want to only plot the GRN genes that are DEG 

bring in counts dataset (warning not normalized) that has the first column as a list of all sig DEGs for every stage comparison 

```{r}
all_stage_DEGs <- read.delim("all_stage_DEGs.txt", sep = "\t")
head(all_stage_DEGs)
dim(all_stage_DEGs) # 12383 
# create vector of stage DEGs 
stage_DEGS <- rownames(all_stage_DEGs)

```

Now I can subset the normalized counts file by only these DEGs 

```{r}
stage_DEG_counts_ortho <- subset(counts_stage_orthogroups, transcript_id %in% stage_DEGS)
dim(stage_DEG_counts_ortho) #12320
# not sure why a few DEGs aren't present 
# also is crazy that almost all genes are DEG in at least one stage comparison 
head(stage_DEG_counts_ortho)

```


```{r}
ortho.GRN <- annot.otho.GRNs[,c(2,4)]

dim(ortho.GRN) #69

EL.ortho.GRN.counts.DEG <- merge(stage_DEG_counts_ortho, ortho.GRN, by= "Orthogroup", sort = TRUE)
EL.ortho.GRN.counts.DEG
dim(EL.ortho.GRN.counts.DEG) # 66 

# so only 3 of the GRN genes are not DEG for some stage
# and only 1 not DEG from all ones that are expressed. Because I already plotted all of them I guess I want to which one is not in both lists?
```


```{r}
GRNcount1 <- EL.ortho.GRN.counts.DEG$common_name
GRNcount2 <- EL.ortho.GRN.counts$ common_name

GRNcount1
GRNcount2

# so Wnt5 is not DEG 
```

want to group graphs as DEG by stage to stage comparisons. There are so many but I think it might be easiest to look at DEG as the linear progression, and not compare like larve to4 cells


Are there GRN genes in the DEG set between eggs and 4 cells?
```{r}
sig_Eggs_4cell <- read.delim("sig_Eggs_4cell.txt", sep = "\t")
sig_Eggs_4cell <- rownames(sig_Eggs_4cell)
EL.ortho.GRN.counts.DEG

fourcell_egg_GRN_counts<- subset(EL.ortho.GRN.counts.DEG, transcript_id %in% sig_Eggs_4cell)
dim(fourcell_egg_GRN_counts)

# 13 of the GRN are DEG for the 4 cells and eggs comparison
```

Plot these at once 

```{r}
fourcell_egg_GRN_counts$common_name

egg_fourcell_DEG_graph <- ggarrange(p58_a_point, FoxQ2_1_point, Eve_point, Sm37_point, Tel_point, Wnt1_point,  Nodal_point, Wnt8_point, HesC_point, Lefty_point, Hox7_point, P19_point, FoxF_point, ncol = 4, nrow= 4)
ggsave("egg_fourcell_DEG_graph3.png", width = 12, height = 10)
```
Note that there is a log scale on the y axis for this!!! 



What about GRN genes DEG between 4 cells and blastula?

```{r}
sig_Blastula_4cell <- read.delim("sig_Blastula_4cell.txt", sep = "\t")
sig_Blastula_4cell <- rownames(sig_Blastula_4cell)
# EL.ortho.GRN.counts.DEG

Blastula_4cell_GRN_counts<- subset(EL.ortho.GRN.counts.DEG, transcript_id %in% sig_Blastula_4cell)
dim(Blastula_4cell_GRN_counts) # 62! almost all of them are significant for between these two. huge transition

```

Split these up into 4 graphs 
```{r}
Blastula_4cell_GRN_counts$common_name


Blastula_4cell_DEG_graph1 <- ggarrange(p58_a_point, Ets1_2_point, FoxQ2_1_point, Erg_point, Ttrspn_19_point, Cara7LA_point, TRINITY_DN48762_c0_g1_not_point, TRINITY_DN5408_c0_g1_not_point, Eve_point, Blimp1_point, Gcm_point, Sm37_point, Otx_point, Pks2_point, Tbr_point, Tel_point, ncol = 4, nrow= 4)
ggsave("Blastula_4cell_DEG_graph1.png", width = 13, height = 10)
```

```{r}
Blastula_4cell_DEG_graph2 <- ggarrange(Sp5_point, Myc_point, Cycpln_point, IrxA_point, Emx_point, Wnt1_point, Wnt16_point, Sm29_point, p58_b_point, Delta_point, FoxG_point, GataC_point, ScratchX_point, Msxl_point, FoxB_point, Endo16_point, ncol = 4, nrow= 4)
ggsave("Blastula_4cell_DEG_graph2.png", width = 13, height = 10)
```

```{r}
Blastula_4cell_DEG_graph3 <- ggarrange(E2f3_point, Nodal_point, Chordin_point, Wnt8_point, Tgif_point, FoxA_point, Alx1_point, Vegf3_point, Sip1_point, Pitx2_point, Nk1_point, SoxE_point, Wnt10_point, Sm50_point, Msp130_point, HesC_point, ncol = 4, nrow= 4)
ggsave("Blastula_4cell_DEG_graph3.png", width = 13, height = 10)
```

```{r}
Blastula_4cell_DEG_graph4 <- ggarrange(C_lectin_point, SoxC_point, Shr2_point, FoxY_point, Ese_point, Gatae_point, Gsc_point, Lefty_point, Six3_point, Bra_point, Hox7_point, Hh_point, P19_point, FoxF_point, ncol = 4, nrow= 4)
ggsave("Blastula_4cell_DEG_graph4.png", width = 13, height = 10)
```






```{r}
sig_Blastula_Gastrula <- read.delim("sig_Blastula_Gastrula.txt", sep = "\t")
sig_Blastula_Gastrula <- rownames(sig_Blastula_Gastrula)
# EL.ortho.GRN.counts.DEG

Blastula_Gastrula_GRN_counts<- subset(EL.ortho.GRN.counts.DEG, transcript_id %in% sig_Blastula_Gastrula)
dim(Blastula_Gastrula_GRN_counts) #  37
```

3 graphs for this, 12, 12 and 13
```{r}
Blastula_Gastrula_GRN_counts$common_name


Blastula_Gastrula_DEG_graph1 <- ggarrange(Ets1_2_point, FoxQ2_1_point, Erg_point, Ttrspn_19_point, Cara7LA_point, TRINITY_DN48762_c0_g1_not_point, TRINITY_DN5408_c0_g1_not_point, Eve_point, Blimp1_point, RhoA_point, Gcm_point, Sm37_point, ncol = 4, nrow= 3)
ggsave("Blastula_Gastrula_DEG_graph1.png", width = 13, height = 10)
```

```{r}
Blastula_Gastrula_DEG_graph2 <- ggarrange(Otx_point, Pks2_point, Tbr_point, Tel_point, Myc_point, Wnt16_point, Sm29_point, GataC_point, ScratchX_point, FoxB_point, Endo16_point, Chordin_point, ncol = 4, nrow= 3)
ggsave("Blastula_Gastrula_DEG_graph2.png", width = 13, height = 10)
```


```{r}
Blastula_Gastrula_DEG_graph3 <- ggarrange(Alx1_point, Vegf3_point, Pitx2_point, SoxE_point, Lox_point, Wnt10_point, FoxC_point, Sm50_point, Msp130_point, Shr2_point, Gsc_point, Hh_point, FoxF_point, ncol = 4, nrow= 4)
ggsave("Blastula_Gastrula_DEG_graph3.png", width = 13, height = 10)
```



```{r}
sig_Gastrula_Larvae <- read.delim("sig_Gastrula_Larvae.txt", sep = "\t")
sig_Gastrula_Larvae <- rownames(sig_Gastrula_Larvae)
# EL.ortho.GRN.counts.DEG

Gastrula_Larvae_GRN_counts<- subset(EL.ortho.GRN.counts.DEG, transcript_id %in% sig_Gastrula_Larvae)
dim(Gastrula_Larvae_GRN_counts) #  37
```

```{r}
Gastrula_Larvae_GRN_counts$common_name


Gastrula_Larvae_DEG_graph1 <- ggarrange(Ets1_2_point, FoxQ2_1_point, Erg_point, Ttrspn_19_point, Cara7LA_point, TRINITY_DN48762_c0_g1_not_point, TRINITY_DN5408_c0_g1_not_point, Eve_point, Blimp1_point, RhoA_point, Gcm_point, Sm37_point, ncol = 4, nrow= 3)
ggsave("Gastrula_Larvae_DEG_graph1.png", width = 13, height = 10)
```


```{r}
Gastrula_Larvae_DEG_graph2 <- ggarrange(Otx_point, Pks2_point, Tbr_point, Tel_point, Myc_point, Wnt16_point, Sm29_point, GataC_point, ScratchX_point, FoxB_point, Endo16_point, Chordin_point, ncol = 4, nrow= 3)
ggsave("Gastrula_Larvae_DEG_graph2.png", width = 13, height = 10)
```

```{r}
Gastrula_Larvae_DEG_graph3 <- ggarrange(Alx1_point, Vegf3_point, Pitx2_point, SoxE_point, Lox_point, Wnt10_point, FoxC_point, Sm50_point, Msp130_point, Shr2_point, Gsc_point, Hh_point, FoxF_point, ncol = 4, nrow= 4)
ggsave("Gastrula_Larvae_DEG_graph3.png", width = 13, height = 10)
```



















