---
title: "HSP-Vis"
author: "Maggie Schedl"
date: "7/1/2020"
output: github_document
---

```{r}
library(DESeq2)
library(ggplot2)
library(pheatmap)
library(RColorBrewer)
library(reshape2)
library(rsconnect)
library(gplots)
library(ashr)
library(dplyr)
library(tidyr)
library(dlookr)
library(tximport)
library(naniar)
library(tidyverse)
library(dplyr)
library(stringr)
library(ggpubr)
```

Read i orthogroups file and do same file manipulations as in annotation script
```{r}
orthologs <- read.delim("LORF_EL90.pep__v__SPU_peptide.tsv", sep = "\t")
head(orthologs)
EL.orthologs <- orthologs %>% 
  select(1,2)
head(EL.orthologs)
# collapse df in to one column multiple rows instead of one row multiple columns
tr.orthogroups <- separate_rows(EL.orthologs, "LORF_EL90.pep", sep = ",", convert = FALSE)
head(tr.orthogroups)
# remove extra characters of .p#s
tr.orthogroups <-mapply(gsub, pattern = ".p1", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# I don't know why this always removes the column names but it does
# remove .p2
tr.orthogroups <-mapply(gsub, pattern = ".p2", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# remove .p3
tr.orthogroups <-mapply(gsub, pattern = ".p3", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# remove .p4
tr.orthogroups <-mapply(gsub, pattern = ".p4", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# remove .p5
tr.orthogroups <-mapply(gsub, pattern = ".p5", replacement = "", tr.orthogroups)
tr.orthogroups <- as.data.frame(tr.orthogroups)
# add column names
colnames(tr.orthogroups) <- c("Orthogroup", "transcript_id")
head(tr.orthogroups)


# remove the isoform notations from this file, because the counts file is only by gene. Its ok to collapse by uniquing the transcript ID because they should be in the same orthogroup 
# I can use . as any character! 
tr.orthogroups.genes <-mapply(gsub, pattern = "_i.", replacement = "", tr.orthogroups)
tr.orthogroups.genes <- as.data.frame(tr.orthogroups.genes)
# there might be random spaces in the file??
tr.orthogroups.genes <-mapply(gsub, pattern = " ", replacement = "", tr.orthogroups.genes)
tr.orthogroups.genes <- as.data.frame(tr.orthogroups.genes)
colnames(tr.orthogroups.genes) <- c("Orthogroup", "transcript_id")
tr.orthogroups.genes.d <- unique(tr.orthogroups.genes)

tr.orthogroups.genes[1:100,]
tr.orthogroups.genes.d[1:100,]
```

Read in the conts file 
This is from the combined model 
```{r}
normalized_counts_dds <- read.delim("normalized_counts_dds.txt", sep = "\t")

colnames(normalized_counts_dds) <- c("transcript_id","29_4cell_rep_1" ,"29_4cell_rep_2" ,"29_4cell_rep_3", "29_blast_rep_1", "29_blast_rep_2" ,"29_blast_rep_3", "29_gast_rep_1",  "29_gast_rep_2",  "29_gast_rep_3", "29_larv_rep_1",  "29_larv_rep_2",  "29_larv_rep_3", "33_4cell_rep_1", "33_4cell_rep_2", "33_4cell_rep_3", "33_blast_rep_1", "33_blast_rep_2", "33_blast_rep_3", "33_gast_rep_1", "33_gast_rep_2",  "33_gast_rep_3",  "33_larv_rep_1",  "33_larv_rep_2",  "33_larv_rep_3", "eggs_rep_1" , "eggs_rep_2",  "eggs_rep_3"  )
head(normalized_counts_dds)
```

```{r}
counts_dds_orthogroups <- merge(normalized_counts_dds, tr.orthogroups.genes.d, by= "transcript_id", sort = TRUE )
head(counts_dds_orthogroups)
dim(normalized_counts_dds) # 13296
dim(counts_dds_orthogroups) # 13226

# not all the ones in the original orthogroups file are present because lowly expressed genes were filtered out in making the normalized count matrix
```


Read in the annotation file that has orthogroups and limited to HSP only, hopefully these commands with grep get all of the HSPs

grep Hsp annot.ortho.SPU.txt > HSP.SPU.txt
```{r}
HSP.SPU <- read.delim("HSP.SPU.txt", sep = "\t", header = FALSE)
colnames(HSP.SPU) <- c("line", "spu_id",   "Orthogroup",  "family_member" ,"common_name", "synonyms",  "best_genbank_hit")
HSP.SPU$line <- NULL
HSP.SPU
dim(HSP.SPU) # 46
```



Separate out the orhtogroup and the commonname columns and then merge by orthogroup to the counts matrix
```{r}
ortho.HSP <- HSP.SPU[,c(2,4)]

dim(ortho.HSP) #18

EL.ortho.HSP.counts <- merge(counts_dds_orthogroups, ortho.HSP, by= "Orthogroup", sort = TRUE)
EL.ortho.HSP.counts
dim(EL.ortho.HSP.counts) #46
# all HSP are present in the counts dataset

Sample_info <- read.csv("el-treatment-data.csv", header=TRUE, sep=",")
```

Plotting the DEG HSPs, so bring in the DEG files to see which ones are in there 
Want to plot only HSP plots that are ones that are significantly differentially expressed in one of the life stages by temperature


Gasturla DEGS
```{r}
gast_DEGs <- read.csv("Gastrula_temp_DEG_annot.csv", header = TRUE)
gast_DEGs
```

Looks like the HSP DEG for this stage is only HSP901,  which is in the family of HSP90


Blastula DEGS
```{r}
blast_DEGs <- read.csv("Blastula_temp_DEG_annot.csv", header = TRUE)
blast_DEGs

```

Hsp702A Hsp701A Hsp701C 	Hsp902a1 Hsp40B


larvae has Hsp40B as well 


Have to separate out  EL.ortho.HSP.counts by the row that is the correct HSP 

Plot Hsp40B Significantly differentially expressed for Larvae and Blastula 
```{r}
# 	Hsp40B
# separate out row number 33
gene<- EL.ortho.HSP.counts[33,]
# 
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
t.gene_samp_info$Temp <- factor(t.gene_samp_info$Temp,
    levels = c("ambient", "high"),ordered = TRUE)
Hsp40B_box <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, fill = Temp)) + 
  geom_boxplot() + ggtitle("Hsp40B") + scale_fill_manual(values=c("skyblue1", "tomato3")) +  theme_linedraw() + geom_dotplot(binaxis='y', stackdir='center', position=position_jitterdodge(0), dotsize = .75)
Hsp40B_box


```

HSP702A DEG for Blastula 
```{r}
# 	Hsp702A
# separate out row number 27
gene<- EL.ortho.HSP.counts[27,]
# 
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
t.gene_samp_info$Temp <- factor(t.gene_samp_info$Temp,
    levels = c("ambient", "high"),ordered = TRUE)
Hsp702A_box <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, fill = Temp)) + 
  geom_boxplot() + ggtitle("Hsp702A") + scale_fill_manual(values=c("skyblue1", "tomato3")) +  theme_linedraw() + geom_dotplot(binaxis='y', stackdir='center', position=position_jitterdodge(0), dotsize = .75)
Hsp702A_box
```


HSP901 DEG for gastrula 
```{r}
# Hsp901
# separate out row number 5
gene<- EL.ortho.HSP.counts[5,]
# 
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
t.gene_samp_info$Temp <- factor(t.gene_samp_info$Temp,
    levels = c("ambient", "high"),ordered = TRUE)
Hsp901_box <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, fill = Temp)) + 
  geom_boxplot() + ggtitle("Hsp901") + scale_fill_manual(values=c("skyblue1", "tomato3")) +  theme_linedraw() + geom_dotplot(binaxis='y', stackdir='center', position=position_jitterdodge(0), dotsize = .75)
Hsp901_box
```

HSP701C DEG for Blastula 
```{r}
# Hsp701C
# separate out row number  3
gene<- EL.ortho.HSP.counts[3,]
# 
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
t.gene_samp_info$Temp <- factor(t.gene_samp_info$Temp,
    levels = c("ambient", "high"),ordered = TRUE)
Hsp701C_box <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, fill = Temp)) + 
  geom_boxplot() + ggtitle("Hsp701C") + scale_fill_manual(values=c("skyblue1", "tomato3")) +  theme_linedraw() + geom_dotplot(binaxis='y', stackdir='center', position=position_jitterdodge(0), dotsize = .75)
Hsp701C_box
```

Hsp701A DEG for Blastula 
```{r}
# Hsp701A
# separate out row number
gene<- EL.ortho.HSP.counts[2,]
# 
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
t.gene_samp_info$Temp <- factor(t.gene_samp_info$Temp,
    levels = c("ambient", "high"),ordered = TRUE)
Hsp701A_box <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, fill = Temp)) + 
  geom_boxplot() + ggtitle("Hsp701A") + scale_fill_manual(values=c("skyblue1", "tomato3")) +  theme_linedraw() + geom_dotplot(binaxis='y', stackdir='center', position=position_jitterdodge(0), dotsize = .75)
Hsp701A_box
```

Hsp902a1 DEG for Blastula 
```{r}
# Hsp902a1
# separate out row number
gene<- EL.ortho.HSP.counts[6,]
# 
gene_samp <- gene[,c(3:29)]
# transpose
t.gene_samp <- as.data.frame(t(gene_samp))
# add in other info for graphing 
t.gene_samp_info <- cbind(Sample_info, t.gene_samp)
rownames(t.gene_samp_info) <- NULL
colnames(t.gene_samp_info) <- c("Sample", "Stage", "Temp", "Normalized_Counts")
t.gene_samp_info$Stage <- factor(t.gene_samp_info$Stage,
    levels = c('eggs','fourcell', "blastula", "gastrula", "larvae"),ordered = TRUE)
t.gene_samp_info$Temp <- factor(t.gene_samp_info$Temp,
    levels = c("ambient", "high"),ordered = TRUE)
Hsp902a1_box <- ggplot(t.gene_samp_info, aes(x=Stage, y=Normalized_Counts, fill = Temp)) + 
  geom_boxplot() + ggtitle("Hsp902a1") + scale_fill_manual(values=c("skyblue1", "tomato3")) +  theme_linedraw() + geom_dotplot(binaxis='y', stackdir='center', position=position_jitterdodge(0), dotsize = .75)
Hsp902a1_box
```


Combine all these plots into one

```{r}

temp.HSP.box <- ggarrange(Hsp902a1_box, Hsp701A_box, Hsp701C_box, Hsp901_box, Hsp702A_box, Hsp40B_box, ncol = 3, nrow = 2)
temp.HSP.box 
ggsave("temp.HSP.box.png", width = 15, height = 7)
```









